### DISCUSSION TITLE
Stage III - Court Detection

### DISCUSSION THREAD

**Discussion by shukkkur** [shukkkur/VolleyVision#7] [Show and tell]
## Welcome! 

By opening this discussion, I invite you 
- to ask questions about this stage of the project
- provide constructive criticism, 
- suggest new or improved methods.

 Your valuable input will help further enhance the accuracy and effectiveness of the tracking system. It will also help me and other users of VolleyVision to learn and grow! 

<h3 align="center"> <strong>Let's collaborate and explore the possibilities of advancing this project together!</strong></h3>

**Comment by fatfryer** [comment]
Have you tried making like a rectangle transpose with the court and randomly selected confident bounds on the edges? We could stabilize the camera. then using some geometry and linear algebra get a great idea on the court demensions that we can even use to remap the players into different angles.
